### ISSUE TITLE
(feature) CSS grid layout

### ISSUE THREAD

**Issue by necolas** [facebook/react-strict-dom#1]
### Describe the feature request

Support [CSS grid layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout) on native. This cannot be polyfilled.

**Comment by TomMarius** [comment]
Yes, please! I'm willing to help out with this. If you need any help, please let me know.

**Comment by TomMarius** [comment]
Yes! I'm willing to help out with this. If you need any help, please let me know.

**Comment by TomMarius** [comment]
Yes, please! Please let me know if you need any help with this, I'm willing to contribute.

**Comment by TomMarius** [comment]
Yes, please! I'm willing to contribute, let me know if you could use my help with this.

**Comment by bayareaunicorn** [comment]
> Yes, please! I'm willing to contribute, let me know if you could use my help with this.

It's open source so anyone can contribute. 
https://github.com/facebook/react-strict-dom/blob/main/CONTRIBUTING.md
https://code.facebook.com/cla

**Comment by necolas** [comment]
Grid layout requires contribution to Yoga, the layout engine for React Native https://github.com/facebook/yoga

There is an existing thread of discussion over the last few years https://github.com/facebook/yoga/issues/867
